index.html

```diff
@@ -64,5 +64,214 @@ <h1 class="header__connect">
         </li>
       </ul>
     </section>
+
+    <!-- /**
+        *! Works Section Here.
+        **/ -->
+
+    <section class="works">
+      <div class="works__cards">
+        <img src="./images/card-1.svg" alt="" />
+        <h1 class="project__title">Tonic</h1>
```

Comment:
- [ ] Kindly consider giving it an `h` tag other than h1 because following the best semantics and best practices, we should only have one h1 heading on each webpage. so consider removing all the `h1` tags from your webpage other than the first one for the main heading in the hero section.